### PR TITLE
Add auto-correct to MixinGrouping

### PR DESCRIPTION
**Pull request by rrosenblum**
I noticed that `Style/MixinGrouping` didn't support auto-correct so I decided to add it. So far I have the auto-correct of `separated` working, and I am trying to see if it will be feasible to add auto-correct for the `grouped` style.

### CODE REVIEW

File: spec/rubocop/cop/style/mixin_grouping_spec.rb [hunk 1]
```diff
@@ -45,14 +45,17 @@
...
 
       context 'with several mixins in one call' do
-        it_behaves_like 'code with offense', <<-END.strip_indent
-          class Foo
-            include Bar, Qux
-          end
-        END
+        it_behaves_like 'code with offense',
+                        ['class Foo',
+                         '  include Bar, Qux',
```
Comment (Drenmi) [inline spec/rubocop/cop/style/mixin_grouping_spec.rb]: As per [this comment](https://github.com/bbatsov/rubocop/issues/3936#issuecomment-280255352), mixins are applied last-to-first when using this style, so the auto-correct should be:

```ruby
include Qux
include Bar
```

This surprised me a bit at first. 🙂

Comment (rrosenblum) [inline spec/rubocop/cop/style/mixin_grouping_spec.rb]: Interesting, I did not know that. Thanks for the link to the comment. I will fix this.

**Comment by rrosenblum** [comment]
@bbatsov this is ready for review.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,10 @@
 * [#4249](https://github.com/bbatsov/rubocop/issues/4249): Handle multiple assignment in `Rails/RelativeDateConstant`. ([@bbatsov][])
 * [#4250](https://github.com/bbatsov/rubocop/issues/4250): Improve a bit the Ruby code detection config. ([@bbatsov][])
 
+### New features
```
Comment (bbatsov) [inline CHANGELOG.md]: There's already such a section the current `master`. Please, rebase.

**Comment by bbatsov** [comment]
Apart from my small remark, the code looks good.

File: CHANGELOG.md (hunk 2 above)
Comment (rrosenblum) [inline CHANGELOG.md]: Whoops, the section didn't exist when I first added it. I have rebased and removed the duplicate section.

**Pull request merged**